README.rst

```diff
@@ -1377,7 +1377,7 @@ An example of ``output_fn`` for the accept type "application/python-pickle" can
 
     import numpy as np
 
-    def output_fn(data, accepts):
+    def output_fn(prediction_result, accepts):
         """An output_fn that dumps a pickled numpy as response"""
         if request_content_type == "application/python-pickle":
             return np.dumps(data)
```

Comment:
should `data` be changed to `prediction_result` here?

Reply:
I did that for consistency, look at the code block below [here](https://github.com/mvsusp/sagemaker-python-sdk/blob/8de091b4dad1023cdf34a9a5e5edf7341e7ee342/README.rst#endpoint-prediction-request-handling)

Reply:
I meant for line 1383 - sorry for the confusion

Reply:
ok